stanford_profile.install

```diff
@@ -47,3 +47,17 @@ function stanford_profile_update_9106() {
     $term->delete();
   }
 }
+
+/**
+ * Removes autogenerated users with roles no longer in use.
+ */
+function stanford_profile_update_9107() {
```

Comment:
I don't think we really need this. The users will still exist without any role. It doesn't really do any harm though. It just adds time to the deployment process.